src/Metal/MTLDevice.cs

```diff
@@ -42,6 +42,28 @@ public static partial class MTLDevice {
 				return system_default;
 			}
 		}
+
+#if MONOMAC
+		//[Mac (10,13, onlyOn64: true), NoiOS, NoWatch, NoTV]
+		//[DllImport (Constants.MetalLibrary)]
+		//static extern IMTLDevice[] MTLCopyAllDevicesWithObserver ([NullAllowed] out NSObject observer, MTLDeviceNotificationHandler handler);
```

Comment:
The whole file is inside a ```#if XAMCORE_2_0 || !MONOMAC``` region, so XM classic shouldn't even be able to see it.  It fails during the step ```PMCS     [mac/mobile-32] Xamarin.Mac.dll```

Build log when attempting another build after it failed the first time: https://gist.github.com/timrisi/ee180c49f793bf7e17d36e0e63005324